Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -113,9 +113,9 @@ function xASL_wrp_Quantify(x, PWI_Path, pathOutputCBF, M0Path, SliceGradientPath
 end
 if ~isfield(x.Q,'T2') || isempty(x.Q.T2)
 	if x.MagneticFieldStrength == 3
-		x.Q.T2 = 180; % default for 3T (ref Jean Chen, MRM 2009)
+		x.Q.T2 = 85; % in ms - default for 3T (ref Johannes Gregori, JMRI 2013)
```

Comment:
Shouldn't we add also the non-3T values, now we're at it?

Reply:
I've added that for T2star and T2blood. 
@BeatrizPadrela can try to locate it for T2-tissue.